syncer/filter_test.go

```diff
@@ -50,39 +52,17 @@ func (s *testFilterSuite) TearDownSuite(c *C) {
 
 func (s *testFilterSuite) TestSkipQueryEvent(c *C) {
 	syncer := &Syncer{}
-	// build in skipDDL
-	skipped, err := syncer.skipQuery(nil, nil, "DROP TEMPORARY TABLE IF EXISTS `foo`.`bar` /* generated by server */")
-	c.Assert(err, IsNil)
-	c.Assert(skipped, IsTrue)
-
-	// system table
-	skipped, err = syncer.skipQuery([]*filter.Table{{Schema: "mysql", Name: "test"}}, nil, "create table mysql.test (id int)")
-	c.Assert(err, IsNil)
-	c.Assert(skipped, IsTrue)
-
 	// test binlog filter
 	filterRules := []*bf.BinlogEventRule{
 		{
-			SchemaPattern: "*",
-			TablePattern:  "",
-			Events:        []bf.EventType{bf.DropTable},
-			SQLPattern:    []string{"^drop\\s+table"},
-			Action:        bf.Ignore,
-		}, {
 			SchemaPattern: "foo*",
 			TablePattern:  "",
 			Events:        []bf.EventType{bf.CreateTable},
 			SQLPattern:    []string{"^create\\s+table"},
-			Action:        bf.Do,
-		}, {
-			SchemaPattern: "foo*",
-			TablePattern:  "bar*",
```

Comment:
please keep the test about of "finer granularity has higher priority"

Reply:
fixed in https://github.com/pingcap/dm/pull/2147/commits/7b53729eea240ff8b9cb6c35385fec57e9ab15ef